tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/IastInstrumentationUnitTests.cs

```diff
@@ -265,7 +265,8 @@ public void TestInstrumentedUnitTests()
 #endif
             SetEnvironmentVariable(ConfigurationKeys.CIVisibility.Enabled, "0"); // without this key, ci visibility is enabled for the samples, which we don't really want
             SetEnvironmentVariable("DD_TRACE_LOG_DIRECTORY", logDirectory);
-            SetEnvironmentVariable("DD_IAST_DEDUPLICATION_ENABLED", "0");
+            SetEnvironmentVariable("DD_IAST_DEDUPLICATION_ENABLED", "1");
+            Console.WriteLine("HELLO FLAVIEN: " + Environment.GetEnvironmentVariable("MONGO_HOST"));
```

Comment:
This looks like debug code. Also deduplication could be explicitly disabled here for something

Reply:
Yes that was only for debugging purpose, it will be removed latter